src/components/orders/OrdersUserDetailsTable/index.tsx

```diff
@@ -22,6 +23,45 @@ const Wrapper = styled(StyledUserDetailsTable)`
   > tbody > tr {
     grid-template-columns: 12rem 7rem repeat(2, minmax(16rem, 1.5fr)) repeat(2, minmax(18rem, 2fr)) 1fr;
   }
+
+  .header-title {
+    display: none;
+  }
+
+  @media ${MEDIA.mobile} {
+    > thead > tr {
+      display: none;
+    }
+    > tbody > tr {
+      border: 0.1rem solid rgb(151 151 184 / 10%);
```

Comment:
There is a shadow on each card:
![image](https://user-images.githubusercontent.com/622217/136074039-89f9d89e-2498-476c-addc-df4ff7a68030.png)
Can you add it? As an example: `box-shadow: 0px 4px 12px rgba(0, 0, 0, 0.18);`